### ISSUE TITLE
version confusion

### ISSUE THREAD

**Issue by binocarlos** [filecoin-project/bacalhau#495]
After running `make devstack` in one pane - I then did:

```bash
make build
sudo cp bin/linux_amd64/bacalhau /usr/local/bin/
bacalhau version
Client Version: v0.1.39-36-g741062b3
Server Version: v0.1.37
```

Why are these two versions different?

I then tried to run a command:

```bash
bacalhau docker run ubuntu echo hello
Error: client version v0.1.37 is newer than server version v0.1.39-36-g741062b3, please ask your network administrator to update Bacalhau
```

`v0.1.37` is not newer than `v0.1.39-36-g741062b3` - but then I realised they are the wrong way around in the error message

**Comment by binocarlos** [comment]
OK - I was being dumb - I was pointing at production so that answers:

> Why are these two versions different?

**Comment by aronchick** [comment]
Right - you didn’t set the environment variables. The other just looks like a bug - I think @philwinder worked on the version skew checker

**Comment by philwinder** [comment]
Hi @binocarlos, yes you're right. You need to set the api-host/port. Something like this:

```
LOG_LEVEL=debug bacalhau --api-port=20000 --api-host=localhost version
```

However, there was a bug in the error message, the client and server versions were incorrectly swapped around. I've fixed this in af47ec43.

Also, the code prevented you from using a real cli version with a development server. I've also fixed this in 6d6a64315434b282874c571391864c045e7746bf. This means any cli version can communicate with a devstack version, so note that if your CLI is old it may not work. Use at your own risk!

And of course, most of the time you probably want to run `go run .` rather than `bacalhau` when doing development.

Thanks, Phil
